static/css/style.css

```diff
@@ -183,6 +183,29 @@ figure{
     display: inherit;
 }
 
+/* Accessibility Page */
+.card-title {
+    font-size: xx-large;
+    text-align: center;
+    padding: 1.5em 0;
+}
+
+.card-text {
+    font-family: "Roboto Condensed", sans-serif;
+    font-size: 16px;
+    padding: 1rem 15%;
+    text-align: center;
+}
+
+.img-fluid {
+    width: 250px;
+}
+
+.text-center {
+    text-align: center !important;
```

Comment:
Consider removing this and replacing it with the utitlity class from Bulma docs to center the text?